src/test/javascript/spec/helpers/mocks/directive/ngbPaginationMocks.module.ts

```diff
@@ -1,23 +1,35 @@
 import { Component, Directive, EventEmitter, Input, NgModule, Output } from '@angular/core';
 
+// eslint-disable-next-line @angular-eslint/directive-selector
 @Directive({ selector: 'ng-template[ngbPaginationLast]' })
+// eslint-disable-next-line @angular-eslint/directive-class-suffix
```

Comment:
Please don't disable the linter here, but rather fix the underlying error (adding the 'Directive' suffix).

Reply:
Done in f14078c614f7f4a4e43952c7bb56f674b71e0bf1